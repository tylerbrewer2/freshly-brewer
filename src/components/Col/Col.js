import React from 'react';

const Col = ({ width, desktopWidth, mobileWidth, tabletWidth, children }) => {
  return (
    <div className={
      `col-xs-${mobileWidth || width} 
      ${tabletWidth && `col-sm-${tabletWidth}`} 
      ${desktopWidth && `col-md-${desktopWidth}`}`
    }>
      {children}
    </div>
  )
}

export default Col;