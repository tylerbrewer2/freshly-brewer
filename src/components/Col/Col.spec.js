import React from 'react';
import renderer from 'react-test-renderer';

import Col from './Col';

it('renders correctly', () => {
  const tree = renderer.create(
    <Col desktopWidth={6}>
      <p>Hello</p>
    </Col>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});