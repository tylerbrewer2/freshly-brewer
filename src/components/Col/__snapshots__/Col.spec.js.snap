exports[`test renders correctly 1`] = `
<div
  className="col-xs-undefined 
      undefined 
      col-md-6">
  <p>
    Hello
  </p>
</div>
`;
